kiwixbuild/dependencies/gumbo.py

```diff
@@ -9,7 +9,7 @@ class Gumbo(Dependency):
     class Source(ReleaseDownload):
         archive = Remotefile(
             "gumbo-parser-0.12.1.tar.gz",
-            "ea46815cb6e5c587cc48321dd6a3c3b9071ca605efb19fc1c1735b2cecc0aac6",
+            "d0fb82d23afc11d52408c63bebc9e34af819cb8458a6f2d85c48c7850412b8a3",
             "https://codeberg.org/gumbo-parser/gumbo-parser/archive/master.tar.gz",
```

Comment:
Now that a new release has been made change the download URL to https://codeberg.org/gumbo-parser/gumbo-parser/archive/0.13.1.tar.gz

Reply:
Done, but from dev.kiwix.org